migrations/strapi-plugin-navigation-3.0.0-no-1-related-id-to-documentid.ts

```diff
@@ -0,0 +1,37 @@
+const SOURCE_TABLE_NAME = 'navigations_items_related';
+const SOURCE_LINK_TABLE_NAME = 'navigations_items_related_links';
+const TARGET_TABLE_NAME = 'navigations_items';
+const RELATED_ITEM_SEPARATOR = '$';
+
+export default {
+  async up(knex) {
+
+    console.log("Navigation plugin :: Migrations :: Backup navigation item related table - START");
+
+    // Get all entries and rewrite directly to the navigation_items table
+    const all = await knex.from(SOURCE_TABLE_NAME).columns('id', 'related_id', 'related_type').select();
+    await Promise.all(
+      all.map(async (item) => {
+        const { related_id, related_type, id: row_id} = item;
+        if (related_id && related_type && !isNaN(parseInt(related_id, 10))) {
```

Comment:
```suggestion
    const all = await knex.from(SOURCE_TABLE_NAME).columns('id', 'related_id', 'related_type').select();
    
    await Promise.all(
      all.map(async (item) => {
        const { related_id, related_type, id: row_id} = item;
        
        if (related_id && related_type && !isNaN(parseInt(related_id, 10))) {
```

I would appreciate spacing, helps with readability.